src/login/oidc/IssuerConfigFetcher.ts

```diff
@@ -154,34 +154,20 @@ export default class IssuerConfigFetcher implements IIssuerConfigFetcher {
   }
 
   async fetchConfig(issuer: URL): Promise<IIssuerConfig> {
-    let issuerConfig: IIssuerConfig = (await this.storageUtility.safeGet(
-      this.getLocalStorageKey(issuer),
-      {
-        schema: issuerConfigSchema
-      }
-    )) as IIssuerConfig;
+    let issuerConfig: IIssuerConfig;
 
-    // If it is not stored locally, fetch it
-    if (!issuerConfig) {
-      const wellKnownUrl = new URL(issuer.toString());
-      wellKnownUrl.set("pathname", "/.well-known/openid-configuration");
-      const issuerConfigRequestBody = await this.fetcher.fetch(wellKnownUrl);
-      // Check the validity of the fetched config
-      try {
-        issuerConfig = this.processConfig(await issuerConfigRequestBody.json());
-      } catch (err) {
-        throw new ConfigurationError(
-          `${issuer.toString()} has an invalid configuration: ${err.message}`
-        );
-      }
+    const wellKnownUrl = new URL(issuer.toString());
+    wellKnownUrl.set("pathname", "/.well-known/openid-configuration");
+    const issuerConfigRequestBody = await this.fetcher.fetch(wellKnownUrl);
+    // Check the validity of the fetched config
+    try {
+      issuerConfig = this.processConfig(await issuerConfigRequestBody.json());
+    } catch (err) {
+      throw new ConfigurationError(
+        `${issuer.toString()} has an invalid configuration: ${err.message}`
```

Comment:
I'd delimit the Issuer variable (I do this everywhere, for all variables in string messages (except for exception messages, as they often contain stack traces that don't warrant delimiting)), and say **_what_** has an invalid config, i.e. I'd replace this with:
`Issuer [${issuer.toString()}] has an invalid configuration: ${err.message}`

Reply:
Fixed in 118cb8eebaca072366cae5f14417d2b712d6dfc6